faunadb-scala/src/main/scala/faunadb/query/package.scala

```diff
@@ -59,6 +59,18 @@ package query {
     case object Delete extends Action(Expr(StringV("delete")))
   }
 
+  /**
+    * Enumeration for casefold operation.
+    */
+  sealed abstract class Normalizer(val expr: Expr)
+  object Normalizer {
+    case object NFD extends Normalizer(Expr(StringV("NFD")))
```

Comment:
I _think_ you don't need the `Expr(StringV(..))` constructor. I believe there are implicit conversions for string to expression.

Reply:
I know, I just followed the same pattern of the other types, `Action` and `TimeUnit`

Reply:
Is it worth changing those too? Just as an improvement to the code?